cranelift/codegen/src/isa/s390x/inst.isle

```diff
@@ -1948,18 +1973,20 @@
 ;; Helpers for register pairs ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 ;; A writable register pair.
-(type WritableRegPair (enum (WritableRegPair (hi WritableReg) (lo WritableReg))))
+(type WritableRegPair extern (enum (WritableRegPair (hi WritableReg) (lo WritableReg))))
```

Comment:
Why the `extern` when it is already fully defined here?

Reply:
This is so that we have the constructors available in the ISLE source, but don't emit the actual declaration.